etc/config/c++.amazon.properties

```diff
@@ -392,6 +392,36 @@ compiler.clang_reflection.semver=(reflection)
 compiler.clang_reflection.options=-std=c++20 -freflection-ts -stdlib=libc++
 compiler.clang_reflection.notification=Experimental Reflection Support; see <a href="https://github.com/cplusplus/reflection-ts" target="_blank" rel="noopener noreferrer">github<sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a>
 
+group.llvm-amdgpu.compilers=llvm-amdgpu-trunk:llvm-amdgpu-40502:llvm-amdgpu-50002:llvm-amdgpu-50103:llvm-amdgpu-50203
+group.llvm-amdgpu.intelAsm=-mllvm --x86-asm-syntax=intel
+group.llvm-amdgpu.options=--gcc-toolchain=/opt/compiler-explorer/gcc-7.2.0
+group.llvm-amdgpu.groupName=llvm-amdgpu x86-64
+group.llvm-amdgpu.instructionSet=amd64
+group.llvm-amdgpu.baseName=x86-64 clang
+group.llvm-amdgpu.isSemVer=true
+group.llvm-amdgpu.compilerType=clang
+group.llvm-amdgpu.unwiseOptions=-march=native
+group.llvm-amdgpu.supportsPVS-Studio=true
+
+compiler.llvm-amdgpu-trunk.exe=/opt/compiler-explorer/llvm-amdgpu-trunk/bin/clang++
+compiler.llvm-amdgpu-trunk.semver=(amd-stg-open)
```

Comment:
I do'nt feel too strongly about this, but calling it `turnk` might help others who don't know what `amd-stg-open` is understand it's a nightly bulit version of the latest version of the code.

Reply:
That's true, although I think giving users the keyword `amd-stg-open` makes it easier for them to find more information about the build. It's also the terminology that the AMD devs are familiar with, which might make it easier to report compiler bugs.

Reply:
THen it's all good. There is going to be so ordering issues with the current semvers of all compilers, but that's something we can fix once this is merged, I'll add myself a reminder to do so :)